### DISCUSSION TITLE
How to : Mermaid support in .md and .html files

### DISCUSSION THREAD

**Discussion by ezzle** [vuejs/vitepress#2522] [Q&A]
Hi

New vitepress user, I need to integrate mermaid  diagram in .md files .

How to ? Does Vitepress support mermaid ?

Best regards

Tshitshi

**Comment by roma-marshall** [comment]


**Reply by brc-dd** [reply]
Same warning as https://github.com/vuejs/vitepress/discussions/2542#discussioncomment-6294585 This one might work though, but it will need some more configuration. Instead one can just use [vitepress-plugin-mermaid](https://github.com/emersonbottero/vitepress-plugin-mermaid).
